io.freetubeapp.FreeTube.yml

```diff
@@ -12,11 +12,16 @@ finish-args:
   - --share=ipc
   - --socket=x11
   - --socket=pulseaudio
-  - --socket=session-bus
   - --share=network
   - --env=TZ=UTC
   - --filesystem=xdg-download
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
   - --own-name=org.mpris.MediaPlayer2.freetube
+  - --talk-name=org.mpris.MediaPlayer2.Player
```

Comment:
I don't think this is an actual name.

Reply:
I think it is, see https://specifications.freedesktop.org/mpris-spec/latest/index.html#Entry-point

Reply:
That is just an interface name. It is not the bus name of any service.

Reply:
Ah yea, you're right. Thanks!